win10-difference.cmd

```diff
@@ -0,0 +1,136 @@
+rem commit message comments:
+
+rem Need to 
+rem - install npm
+rem - install curl
+rem - install more packages
+rem - curl need to use double quote in the curl parameters
+rem - curl need to escape double quote in json string
+
+rem install curl exe
+rem install npm (node.js)
+
+rem update npm 
+
+npm i npm
+
+rem install all missing modules
+
+npm install express
+
+npm install body-parser
+
+npm install swagger-ui-express
+
+npm install ramda
+
+npm install elliptic
+
+npm install es6-error
+
+npm install statuses
+
+npm install timeago.js
+
+npm install fs-extra
+
+npm install threads
+
+npm install superagent
+
+npm install yargs
+
+npm install cli-color
+
+rem starting only one node (need to cancel if start > 1 node)
+
+node bin/naivecoin.js
+
+rem 2018-03-11T14:13:56.779Z - info - 1: Starting node 1
```

Comment:
Why are these lines here? (lines showing the log of naivecoin)

Reply:
I assume this is for learning and one thing I find it is very hard to learn if I do not have a log showing the exact response.  But may be it can be in a separate file e.g. windows-experience-log.txt

I would try to do the above after I try the naivecoin on Mac as well to complete the whole exercise.  

(BTW, I noted there is another question on transfer address validity; is the wallet check the address and/or is there a blockchain explorer.  It would be my second task in this project.)

Reply:
after doing this
    docker-compose up 
work for docker image with 3 nodes and even better it is 0.0.0.0 and hence no need to change the localhost to my ip address.  One thing is that the node is all run under =same 3001 and hence only this the wallet is maintained.
    
and that image work with mac node as well

The only left is ubuntu and hence

    docker run -v ~/Downloads/naivecoin/:/naivecoin -ti ubuntu /bin/sh

I test and the command with " - ' and escape " work for curl.  However, under docker ubuntu mode, the following error message:

> 
# apt-get update
# apt-get install nodejs

# nodejs bin/naivecoin.js  -a 192.168.1.8 -p 3003 --name 3 --peers http://192.168.1.8:3001
/naivecoin/lib/naivecoin.js:19
    let blockchain = new Blockchain(name);
    ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/naivecoin/bin/naivecoin.js:2:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
#

Reply:
A lot to respond here, hehe. Please give me some time to respond. Talk to you soon.